src/screens/CreateTokenName.js

```diff
@@ -46,11 +48,19 @@ class CreateTokenName extends React.Component {
               onChangeText={this.onNameChange}
               value={this.state.name}
             />
-            <NewHathorButton
-              title='Next'
-              disabled={!this.state.name}
-              onPress={this.onButtonPress}
-            />
+            <View>
+              <InfoBox
+                items={[
+                  <Text>Token name should be the full name of the new token you are creating</Text>,
+                  <Italic>E.g. Hathor</Italic>
```

Comment:
I would use another example. Something with two words and a space. Maybe "My Custom Token" or "Hathor Network".

Reply:
The problem is that you're leading the person to use that model. I prefer without spaces: 'YanCoin' or 'MyCustomToken'. I used a single word to make sure we don't prime the user with a preference.

We can also give 2 examples, with and without space.

Reply:
Seems good. Examples: "MyCustomToken", "My Custom Token", etc.

Reply:
Done